### ISSUE TITLE
[BUG] InteractiveEngineInternalError: Create interactive instance failed.

### ISSUE THREAD

**Issue by Song-xx** [alibaba/GraphScope#2080]
I am learning the tutorial 01, installed the graphscope, it reports errors when I run:
interactive = graphscope.gremlin(graph),
the error message is :
```bash
InteractiveEngineInternalError            Traceback (most recent call last)
/Users/xxx/graph/GraphScope/tutorials/zh/01_node_classification_on_citation_network.ipynb Cell 6 in <cell line: 2>()
      1  # Get the entrypoint for submitting Gremlin queries on graph g.
----> 2 interactive = graphscope.gremlin(graph)
      4 # Count the number of papers two authors (with id 2 and 4307) have co-authored.
      5 papers = interactive.execute(
                   "g.V().has('author', 'id', 2).out('writes').where(__.in('writes').has('id', 4307)).count()"
            ).one()

File ~/Library/Python/3.8/lib/python/site-packages/graphscope/client/session.py:1552, in gremlin(graph, engine_params)
   1550 if _default_session_stack.is_cleared():
   1551     raise RuntimeError("No default session found.")
-> 1552 return get_default_session().gremlin(graph, engine_params)

File ~/Library/Python/3.8/lib/python/site-packages/graphscope/client/utils.py:357, in set_defaults.<locals>.decorator.<locals>.wrapper(*args, **kwargs)
    354 assert len(original_defaults) == len(new_defaults), "set defaults failed"
    355 func.__defaults__ = tuple(new_defaults)
--> 357 return_value = func(*args, **kwargs)
    359 # Restore original defaults.
    360 func.__defaults__ = original_defaults

File ~/Library/Python/3.8/lib/python/site-packages/graphscope/client/session.py:1150, in Session.gremlin(self, graph, engine_params)
   1148     return interactive_query
   1149 if interactive_query.status == InteractiveQueryStatus.Failed:
...
   1151 # Initializing.
   1152 # while True is ok, as the status is either running or failed eventually after timeout.
   1153 while True:

InteractiveEngineInternalError: Create interactive instance failed.
```

**Environment (please complete the following information):**
 - GraphScope version: 0.14.0
 - OS: mac m1 ,MacOS 12.5
 - Kubernetes Version:
% kubectl version
I run kuberctl version,it reports:
```bash
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.0", GitCommit:"a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2", GitTreeState:"clean", BuildDate:"2022-08-23T17:44:59Z", GoVersion:"go1.19", Compiler:"gc", Platform:"darwin/arm64"}
Kustomize Version: v4.5.7
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

**Comment by lidongze0629** [comment]
@Song-xx Could you please give more log output of the GIE engine here? The log directory: `/var/log/graphscope`

**Comment by Song-xx** [comment]
> 

stranglely "ls: /var/log/graphscope: No such file or directory"

**Comment by Song-xx** [comment]
> @Song-xx Could you please give more log output of the GIE engine here? The log directory: `/var/log/graphscope`


stranglely "ls: /var/log/graphscope: No such file or directory"

**Comment by sighingnow** [comment]
> > @Song-xx Could you please give more log output of the GIE engine here? The log directory: `/var/log/graphscope`
> 
> stranglely "ls: /var/log/graphscope: No such file or directory"

Try `~/.graphscope/log`.

**Comment by sighingnow** [comment]
Closing as we the codebase where the error occurs has been refactored and please upgrade to our latest release (v0.16.0). We no longer have bandwidth to maintain the old versions.

**Comment by Song-xx** [comment]
> > > @Song-xx Could you please give more log output of the GIE engine here? The log directory: `/var/log/graphscope`
> > 
> > 
> > stranglely "ls: /var/log/graphscope: No such file or directory"
> 
> Try `~/.local/log/graphscope`.

I have solved this problem by using graphscope==0.16.0,thanks
